services/notifications/setup.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import re
+import sys
+from pathlib import Path
+
+from setuptools import find_packages, setup
+
+
+def read_reqs(reqs_path: Path) -> set[str]:
```

Comment:
this function contains a scary regex, I guess there should be a comment above explaining what it does or it should be factored out into some library / package where via its placement this is clear

Reply:
I think you cannot export it in a common package since this runs when your write the equivalent `pip install .`
We have this pattern all across the projects and the name is pretty self expalnatory.

Reply:
ok no problem